.github/workflows/cmake.yml

```diff
@@ -0,0 +1,54 @@
+name: CMake
+
+# When creating a new workflow in GitHub’s action builder the default trigger is the push event. We want to extend this to push and pull request events.
+on: [push, pull_request]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # The branch, tag or SHA to checkout. When checking out the repository that
+        # triggered a workflow, this defaults to the reference or SHA for that event.
+        # Otherwise, uses the default branch.
+        ref: 'latest'
+        #ref: '1.2.0'
```

Comment:
If I'm reading it right you can simply remove this to get the build to work as intended.

Reply:
After removing the `with.ref` for checkout, the build fails with `error: expected expression sgGameInitInfo.versionPatch = PROJECT_VERSION_PATCH;`

A quick search shows that others working on the project have had similar problems. It seems we may need to provide the version manually to CMake since git isn't doing it properly?